ERROR - 2024-07-25 04:35:46 --> mysqli_sql_exception: Unknown column '_' in 'field list' in C:\xampp\htdocs\ampy\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ampy\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\ampy\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\ampy\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\ampy\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\ampy\app\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ampy\system\CodeIgniter.php(933): App\Controllers\Home->addData()
#6 C:\xampp\htdocs\ampy\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\ampy\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ampy\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ampy\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ampy\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-07-25 04:35:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column '_' in 'field list'
[Method: GET, Route: Home/addData]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 2 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-25 04:35:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column '_' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 4 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-25 04:35:46 --> [Caused by] mysqli_sql_exception: Unknown column '_' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 5 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-25 04:36:54 --> mysqli_sql_exception: Unknown column '_' in 'field list' in C:\xampp\htdocs\ampy\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ampy\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\ampy\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\ampy\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\ampy\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\ampy\app\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ampy\system\CodeIgniter.php(933): App\Controllers\Home->addData()
#6 C:\xampp\htdocs\ampy\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\ampy\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ampy\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ampy\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ampy\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-07-25 04:36:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column '_' in 'field list'
[Method: GET, Route: Home/addData]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 2 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-25 04:36:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column '_' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 4 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-25 04:36:54 --> [Caused by] mysqli_sql_exception: Unknown column '_' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `datatable` (`_`) VALUES (\'1721882115022\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 5 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-25 04:41:56 --> mysqli_sql_exception: Unknown column '_' in 'field list' in C:\xampp\htdocs\ampy\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\ampy\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\ampy\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\ampy\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\ampy\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\ampy\app\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ampy\system\CodeIgniter.php(933): App\Controllers\Home->addData()
#6 C:\xampp\htdocs\ampy\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\ampy\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\ampy\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\ampy\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\ampy\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-07-25 04:41:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column '_' in 'field list'
[Method: GET, Route: Home/addData]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 2 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-25 04:41:56 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column '_' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `datatable` (`_`) VALUES (\'1721882115023\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `datatable` (`_`) VALUES (\'1721882115023\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 4 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-25 04:41:56 --> [Caused by] mysqli_sql_exception: Unknown column '_' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `datatable` (`_`) VALUES (\'1721882115023\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `datatable` (`_`) VALUES (\'1721882115023\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `datatable` (`_`) VALUES (\'1721882115023\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `datatable` (`_`) VALUES (:_:)', [...], false)
 5 APPPATH\Controllers\Home.php(38): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->addData()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
